tests/test.ts

```diff
@@ -78,9 +81,12 @@ describe('Test clasp create function', () => {
     );
     expect(result.stderr).to.contain('Project file (.clasp.json) already exists.');
   });
+  after(() => {
```

Comment:
Can you try extracting this after method into a common function to reduce duplicated code?